ui/src/components/DocumentSizeCard.jsx

```diff
@@ -10,7 +10,9 @@ function DocumentSizeCard(props) {
           ? "selected-card w-100"
           : "w-100 scale-down"
       }
-      onClick={() => props.onClick(props.name)}
+      onClick={() =>
+        props.onClick !== undefined ? props.onClick(props.name) : ""
```

Comment:
Returning empty string looks weird. It would be better to return `undefined`. Or just use a simple if statement and don't return anything.